pymilo/exceptions/deserialize_exception.py

```diff
@@ -1,15 +1,42 @@
+# -*- coding: utf-8 -*-
+"""PyMilo Deserialization Exception."""
 from enum import Enum
 from .pymilo_exception import PymiloException
 
 
 class DeSerilaizatoinErrorTypes(Enum):
+    """An enum class to determine the type of deserialization errors."""
+    
     CORRUPTED_JSON_FILE = 1
     INVALID_MODEL = 2
     VALID_MODEL_INVALID_INTERNAL_STRUCTURE = 3
 
 
 class PymiloDeserializationException(PymiloException):
+    """
+    PymiloDeserializationException handle exceptions associated with Deserializations.
```

Comment:
Just for the consistency, we may need to change it to:
```
Handle exceptions associated with Deserializations.
```

Reply:
applied.